src/Login/redux/actions/LoginAction.js

```diff
@@ -0,0 +1,29 @@
+import { toast } from 'react-toastify';
+import axios from 'axios';
+import { loginTypes } from '../types';
+import { loginUrl } from '../loginUrls';
+
+const loginAction = (userdata, props) => (dispatch) => {
+  dispatch({
+    type: loginTypes.IS_LOGGING,
```

Comment:
Provide a better and more meaningful name to the constant `IS_LOGGING`?? In this context `IS_LOGGING` doesn't make sense. Remember you're writing code that will also be read by your team mates and yourself in the future.

Reply:
Kindly note l have worked on this as well @AGMETEOR  thank you